app/assets/javascripts/app.coffee.erb

```diff
@@ -0,0 +1,68 @@
+receta = angular.module('receta',[
+  'templates',
+  'ngRoute',
+  'controllers',
+])
+
+receta.config([ '$routeProvider',
+  ($routeProvider)->
+    $routeProvider
+    .when('/',
+      templateUrl: "<%= asset_path('index.html') %>"
+      controller: 'PcbuilderController'
+    )
+])
+
+components = [
```

Comment:
Yes this needs to be replaced with data of the mongo api when it's done. But for now this is fine I think?